DriverDataUI/src/main/java/com/drivequant/drivekit/ui/mysynthesis/component/communitycard/MySynthesisGaugeView.kt

```diff
@@ -0,0 +1,329 @@
+                for (j in i + 1 until size) {
+                    val otherTextView = this.levelTextViews[j]
+                    if (leftTextView.right + margin >= otherTextView.left) {
+                        otherTextView.visibility = View.INVISIBLE
+                    } else {
+                        break
+                    }
+                }
+            }
+            val rightTextView = this.levelTextViews[size - 1 - i]
+            if (rightTextView.visibility == View.VISIBLE) {
+                for (j in size - 1 - i - 1 downTo 0) {
+                    val otherTextView = this.levelTextViews[j]
+                    if (rightTextView.left - margin <= otherTextView.right) {
+                        otherTextView.visibility = View.INVISIBLE
+                    } else {
+                        break
+                    }
+                }
+            }
+        }
+    }
+
+    fun configure(viewModel: MySynthesisGaugeViewModel) {
+        this.viewModel = viewModel
+        viewModel.onUpdateCallback = this::update
+        update()
+    }
+
+    private fun update() {
+        this.scoreDescription.text = (this.viewModel?.scoreLevel?.getScoreLevelDescription() ?: R.string.dk_driverdata_mysynthesis_can_not_be_evaluated).let { context.getString(it) }
+        this.level0TextView.text = getFormattedLevelValue(0)
+        this.level1TextView.text = getFormattedLevelValue(1)
+        this.level2TextView.text = getFormattedLevelValue(2)
+        this.level3TextView.text = getFormattedLevelValue(3)
+        this.level4TextView.text = getFormattedLevelValue(4)
+        this.level5TextView.text = getFormattedLevelValue(5)
+        this.level6TextView.text = getFormattedLevelValue(6)
+        this.level7TextView.text = getFormattedLevelValue(7)
+
+        this.communityMinValueTextView.text = getFormattedValue(this.viewModel?.communityMinScore)
+        this.communityMeanValueTextView.text = getFormattedValue(this.viewModel?.communityMeanScore)
+        this.communityMaxValueTextView.text = getFormattedValue(this.viewModel?.communityMaxScore)
+
+        this.communityMinTextView.text = context.getString(R.string.dk_driverdata_mysynthesis_minimum)
+        this.communityMeanTextView.text = context.getString(R.string.dk_driverdata_mysynthesis_average)
+        this.communityMaxTextView.text = context.getString(R.string.dk_driverdata_mysynthesis_maximum)
+
+        updateLayout()
+    }
+
+    private fun getLevelValue(index: Int): Double? = this.viewModel?.getLevelValue(index)
+    private fun getFormattedLevelValue(index: Int): String? = getFormattedValue(getLevelValue(index))
+    private fun getFormattedValue(value: Double?): String? = value?.format(1)
+
+    private fun configure() {
+        val synthesisColor = getColor(R.color.dkMySynthesisColor)
+        ImageViewCompat.setImageTintList(this.scoreDescriptionIcon, ColorStateList.valueOf(DriveKitUI.colors.secondaryColor()))
```

Comment:
On doit pouvoir se passer d'`ImageViewCompat` car c'est pour les devices en Android 5 et moins : 

`this.scoreDescriptionIcon.imageTintList = ColorStateList.valueOf(DriveKitUI.colors.secondaryColor())`